sample-android/src/main/java/io/apptik/jus/samples/DetailFragment.java

```diff
@@ -0,0 +1,69 @@
+package io.apptik.jus.samples;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import io.apptik.comm.jus.ui.NetworkImageView;
+import io.apptik.json.JsonObject;
+import rx.Subscription;
+
+
+public class DetailFragment extends Fragment {
+	private TextView itemId;
+	private TextView itemTitle;
+	private NetworkImageView itemPic;
+	Subscription subscription;
+
+	protected void updateData(JsonObject item) {
+		itemId.setText(item.getString("id"));
+		itemTitle.setText(item.getString("title"));
+		itemPic.setImageUrl(item.getString("mediumUrl"), MyJus.imageLoader());
+	}
+
+
+	public DetailFragment() {
+	}
+
+	public static DetailFragment newInstance() {
+		DetailFragment fragment = new DetailFragment();
+		return fragment;
+	}
+
+	@Override
+	public void onDetach() {
+		super.onDetach();
+		if(subscription!=null) {
+			subscription.unsubscribe();
```

Comment:
this happens after the view is destroyed, which could lead to a messy stuff as it can be recreated before that. what about doing it on pause/resume or smth like that ... fragments suck! :)